app/templates/views/download.html

```diff
@@ -10,11 +10,11 @@
   </p>
 
   <p class="govuk-body">
-    Check the email that {{ service_name or '[SERVICE_NAME]'}} sent you for more information.
+    <a href="{{download_link}}" download class="govuk-link" rel="noreferrer">Download this {{ file_type }} ({{ file_size }}) to your device</a>
   </p>
 
   <p class="govuk-body">
-    <a href="{{download_link}}" download class="govuk-link" rel="noreferrer">Download this {{ file_type }} ({{ file_size }}) to your device</a>
+    This file is available until {{file_expiry_date}}.
```

Comment:
Sorry to flag this late - I've just thought about an edge case: it's possible someone could see this line _after_ the expiry date because the file hasn't _actually_ been deleted yet.

Should we have some special content in that case? Something like "This file will be deleted soon." If we go this route we'll need to flag Karl/Saima to get thoughts on content.

Alternatively - and perhaps a better option - is that we could 404 if the current datetime is beyond the `available_until` date? That way we're handling this edge case, and more definitively respecting the service's retention period. 🤔

Reply:
I like "this file will be deleted soon" approach, I will ask @karlchillmaid and @saimaghafoor what they think 👍🏼